core/lib/mno_enterprise/impac_client.rb

```diff
@@ -0,0 +1,19 @@
+module MnoEnterprise
+  class ImpacClient
+    include HTTParty
+
+    def self.host
+      "#{Settings.impac.protocol}://#{Settings.impac.host}"
+    end
+
+    def self.endpoint_url(endpoint, params)
+      "#{host}#{endpoint}?#{params.to_query}"
```

Comment:
Maybe use `"#{File.join(host,endpoint)}?#{params.to_query}"` to avoid problem with `/`